sensing/imu_corrector/src/gyro_bias_estimator.cpp

```diff
@@ -22,59 +29,138 @@ GyroBiasEstimator::GyroBiasEstimator()
   angular_velocity_offset_x_(declare_parameter<double>("angular_velocity_offset_x")),
   angular_velocity_offset_y_(declare_parameter<double>("angular_velocity_offset_y")),
   angular_velocity_offset_z_(declare_parameter<double>("angular_velocity_offset_z")),
+  timer_callback_interval_sec_(declare_parameter<double>("timer_callback_interval_sec")),
+  straight_motion_ang_vel_upper_limit_(
+    declare_parameter<double>("straight_motion_ang_vel_upper_limit")),
   updater_(this),
   gyro_bias_(std::nullopt)
 {
   updater_.setHardwareID(get_name());
   updater_.add("gyro_bias_validator", this, &GyroBiasEstimator::update_diagnostics);
-
-  const double velocity_threshold = declare_parameter<double>("velocity_threshold");
-  const double timestamp_threshold = declare_parameter<double>("timestamp_threshold");
   const size_t data_num_threshold =
     static_cast<size_t>(declare_parameter<int>("data_num_threshold"));
-  gyro_bias_estimation_module_ = std::make_unique<GyroBiasEstimationModule>(
-    velocity_threshold, timestamp_threshold, data_num_threshold);
+
+  gyro_bias_estimation_module_ = std::make_unique<GyroBiasEstimationModule>(data_num_threshold);
 
   imu_sub_ = create_subscription<Imu>(
     "~/input/imu_raw", rclcpp::SensorDataQoS(),
     [this](const Imu::ConstSharedPtr msg) { callback_imu(msg); });
-  twist_sub_ = create_subscription<TwistWithCovarianceStamped>(
-    "~/input/twist", rclcpp::SensorDataQoS(),
-    [this](const TwistWithCovarianceStamped::ConstSharedPtr msg) { callback_twist(msg); });
-
+  pose_sub_ = create_subscription<PoseWithCovarianceStamped>(
+    "~/input/pose", rclcpp::SensorDataQoS(),
+    [this](const PoseWithCovarianceStamped::ConstSharedPtr msg) { callback_pose(msg); });
   gyro_bias_pub_ = create_publisher<Vector3Stamped>("~/output/gyro_bias", rclcpp::SensorDataQoS());
```

Comment:
Please publish the estimated gyro_bias as a topic as well

Reply:
Thank you.
It was a mistake to delete. I have restored it.
https://github.com/autowarefoundation/autoware.universe/pull/5259/commits/b762f909210755ae98e9f16a3380f0bff5d76188